settings.gradle

```diff
@@ -1 +1 @@
-include ':example', ':testharness', ':sdk', ':examplelib'
```

Comment:
Running `./gradlew clean :mazerunner:assemble --profile` currently completes in ~15s. I'd agree that we should keep an eye on how long it takes to build as we add more scenarios, but think that there's currently still a benefit to building this on CI as it will detect any compilation errors.

Reply:
On CI, sure. I thought it was odd/long to run `gradle build` locally and automatically run the examples.